planning/autoware_path_generator/param/path_generator_parameters.yaml

```diff
@@ -0,0 +1,21 @@
+path_generator:
```

Comment:
I guess the directory should be not `param/` but `config/`.

Reply:
This file does not provide a configuration example, but rather a description for code generation by generate_parameter_library. Isn't it confusing to rename the directory to `config`?

Reply:
I got it. Is there any discussion on the GitHub related to this configuration example?
If there is no guideline about this, let's keep it as it is.

Reply:
Ota-san asked me the same question and we ended up putting it in `param` at that time: https://github.com/autowarefoundation/autoware.universe/pull/8719#discussion_r1741335711